.github/workflows/reset-label.yml

```diff
@@ -20,7 +20,9 @@ jobs:
       pull-requests: write
 
     if: |
-      github.event.label.name == inputs.label
+      true
+      && github.event_name == 'label'
```

Comment:
I think this condition is wrong.
When we label a PR the event_name is `pull_request`. 
The action is labeled:
```
&& github.event.action == 'labeled'
```

Reply:
Already fixed